ph5/utilities/ph5validate.py

```diff
@@ -474,12 +473,13 @@ def check_station_completeness(self, station):
         if sensor_serial is None:
             warning.append("Sensor serial number is missing.")
 
-        if das_serial not in self.ph5.Das_t:
-            error.append("No data found for station {0}. "
-                         "You may need to reload the raw "
-                         "data for this station."
-                         .format(str(station_id)))
         try:
+            self.ph5.read_das_t(das_serial, reread=False)
+            if das_serial not in self.ph5.Das_t:
+                error.append("No data found for das serial number {0}. "
+                             "You may need to reload the raw "
+                             "data for this station."
+                             .format(str(das_serial)))
```

Comment:
Perhaps this should be in it's own try.
With this and the original try having specific exceptions expected.